index.bs

```diff
@@ -754,6 +754,28 @@ the maximum number of attributions for a ([=attribution rate-limit record/source
 [=attribution rate-limit record/attribution destination=],
 [=attribution rate-limit record/reporting endpoint=]) per [=attribution rate-limit window=].
 
+<h3 dfn id="should-block-attribution-for-reporting-origin-limit">Should attribution be blocked by reporting-origin limit</h3>
+
+Given an [=attribution trigger=] |trigger| and [=attribution source=] |sourceToAttribute|:
+
+1. Let |sourceSite| be the result of [=obtain a site|obtaining a site=] from |sourceToAttribute|'s
+    [=attribution source/source origin=].
+1. Let |matchingRateLimitRecords| be all entries in the [=attribution rate-limit cache=] where all of the following are true:
```

Comment:
nit: wdyt about combining this with the set construction for brevity

1.  Let |distinctReportingOrigins| be a new empty [=ordered set=].
1. [=set/Append=] all entries in the [=attribution rate-limit cache=] where all of the following are true to |distinctReportingOrigins|:
....